roles/tackle/templates/deployment-hub.yml.j2

```diff
@@ -39,6 +39,11 @@ spec:
         app.kubernetes.io/part-of: {{ app_name }}
         app: {{ app_name }}
         role: {{ hub_service_name }}
+{% if feature_auth_required|bool %}
+{% if app_profile == 'mta' %}
+        keycloak_db_secret_name: {{ keycloak_db_secret.env | k8s_config_resource_name }}
```

Comment:
Curious what would this label be used for?

Reply:
There has always been the issue that you update an underlying config for a deployment, be that a secret, configmap, the entire authentication system, that we won't be picked up until the pod restarted. The way we handled this for mig-controller in MTC when we found them was to add these. This generates a label:
```
keycloak_db_secret_name: keycloak-db-secret-c74gh8bdd5
```
The last bit changes based on the contents of the calculated contents of the file. We use the template so status, etc. doesn't affect it. Anytime the operator would update the file, the generated name changes, the label is updated, the pod restarts.

I believe fabianvf taught us this. You can see more examples:
https://github.com/migtools/mig-operator/blob/master/roles/migrationcontroller/templates/controller.yml.j2#L45-L47